source/features/next-scheduled-github-action.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {parseCron} from '@cheap-glitch/mi-cron';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL, getRepoGQL} from '../github-helpers';
+
+const getActionsSchedules = cache.function(async (): Promise<Record<string, string> | false> => {
+	const {repository: {object: {entries: actions}}} = await api.v4(`
```

Comment:
These are not actions but workflows. This applies to every place where we call them `actions` (except in the name because that's the official name of "GitHub Actions")
```suggestion
	const {repository: {object: {entries: workflow}}} = await api.v4(`
```